docs/backends.rst

```diff
@@ -0,0 +1,211 @@
+===================================
+ Proctoring backend implementation
+===================================
+
+Proctoring providers (PS) who wish to integrate with Open edX should implement a `REST API`_ and a thin `Python wrapper`_, as described below.
+
+REST API
+--------
+
+Implement the following endpoints. In order to authenticate requests from the Open edX server, the PS backend should
+enable Oauth 2 authentication using JWT_. The Open edX installation will be configured with a client key and a client secret provided by the PS and will authenticate each request to the PS.
+
+To obtain a JWT_ token, the PS system must make an Oauth2 request to ``/oauth2/access_token``.
+
+SSL is required for all (production environment) requests.
+
+All requests and responses in this API are formatted as JSON objects.
+
+
+Proctoring System configuration endpoint
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+    /v1/config/
```

Comment:
can we have these be a sub-entity to `config` called `rules`? I feel like we're using that framing a lot in conversation around this issue, and it's possible there may be non-rule-oriented configs for other providers in the future

Reply:
Do you mean rename ```config``` to ```rules``` in the JSON response?